compiler/damlc/daml-doc/src/DA/Daml/Doc/Render/Rst.hs

```diff
@@ -37,10 +37,16 @@ renderRstText env = \case
     RenderConcat ts -> mconcatMap (renderRstText env) ts
     RenderPlain text -> text
     RenderStrong text -> T.concat ["**", text, "**"]
-    RenderLink anchor text ->
-        case lookupAnchor env anchor of
+    RenderLink ref text ->
+        case lookupReference env ref of
             Nothing -> text
-            Just _ -> T.concat ["`", text, " <", unAnchor anchor, "_>`_"]
+            Just anchorLoc@(External _) ->
+                T.concat
+                    ["`", text, " <"
+                    , anchorHyperlink anchorLoc (referenceAnchor ref)
+                    , ">`_"]
```

Comment:
We seem to have lost a `_` before the closing `>`. Is that intentional?

Reply:
Yep! That's RST for ya! http://docutils.sourceforge.net/docs/user/rst/quickref.html#external-hyperlink-targets

Reply:
RTS is great …